### PR TITLE
[FLIN-91] Create GitHub guide on how to commit the work done and how to reviw it

### PR DESCRIPTION
**Pull request by AZANK7173**
---
name: Pull Request
title: '[FLIN-91] Create GitHub guide on how to commit the work done and how to reviw it'
---

## Describe your changes
Update to the README what to do reagrding commits and issue creation 

## Github issue this pull requests addresses
Closes issue #25 

## Checklist before requesting a review
- [ ] I have performed a self-review of my code

[FLIN-91]: https://flinkfood.atlassian.net/browse/FLIN-91?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,10 +79,11 @@ docker exec -it $(docker ps --filter name=jobmanager --format={{.ID}}) /bin/sh
...
 Contributions to FlinkFood are welcome! Follow these steps to contribute:
 
-1. Create a new branch for your feature from the develop branch (`git checkout -b JRA-123-<branch-name>`)
-2. Make changes and commit (`git commit -m 'JRA-123 <commit message>'`)
-3. Push to the branch (`git push origin JRA-123-<branch-name>`)
-4. Create a pull request into the develop branch
+1. Create a new branch for your feature from the develop branch (`git checkout -b "(feature | hotfix | release | docs) / #of_task - really_brief_description"`) no spaces.
+2. Make changes, re-run the docker containers to test and commit (`git commit -m 'name_of_created_branch'`). In case you have done the work along-side someone (Parallel computing), you can commit with them as co-authors (how to do it: [here](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors))
+3. Push to the branch (`git push origin 'name_of_created_branch'`)
+4. Create an issue on the GitHub page, following the pre defined template, make sure to add the reviewers of the work as the co-collaborators assigned on the Jira board
```
Comment (PolimiGiovanniArriciati) [inline README.md:85]: I'd remove the issue task

**Review by PolimiGiovanniArriciati — CHANGES_REQUESTED** [review]

**Comment by AZANK7173** [comment]
@PolimiGiovanniArriciati Done :) removed issues 👍

File: README.md [hunk 2]
```diff
@@ -80,20 +80,13 @@ docker exec -it $(docker ps --filter name=jobmanager --format={{.ID}}) /bin/sh
...
 Please follow the [Git flow workflow](https://www.gitkraken.com/learn/git/git-flow) and adhere to the established code style.
 
-To make sure that the issues, commits and PRs are linked to the Jira issues. Make sure to follow these rules:
-
-### Issue naming
-
-Please be sure to name your issue (if the ID of the Jira task is FLIN-123):
-`[FLIN-123] Name of issue`
-This will automatically link the issue with your issue on Jira
+To make sure that the commits and PRs are linked to the Jira issues. Make sure to follow these rules:
```
Comment (PolimiGiovanniArriciati) [inline README.md:89]: this does not belong here anymore

File: README.md [hunk 3]
```diff
@@ -80,20 +80,13 @@ docker exec -it $(docker ps --filter name=jobmanager --format={{.ID}}) /bin/sh
 Contributions to FlinkFood are welcome! Follow these steps to contribute:
 
 1. Create a new branch for your feature from the develop branch (`git checkout -b "(feature | hotfix | release | docs) / #of_task - really_brief_description"`) no spaces.
-2. Make changes, re-run the docker containers to test and commit (`git commit -m 'name_of_created_branch'`). In case you have done the work along-side someone (Parallel computing), you can commit with them as co-authors (how to do it: [here](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors))
+2. Make changes, re-run the docker containers to test and commit (`git commit -m 'number_of_task/simple_commit_message'`, eg: `git commit -m '91/update_README.md'`). In case you have done the work along-side someone (Parallel computing), you can commit with them as co-authors (how to do it: [here](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors))
```
Comment (PolimiGiovanniArriciati) [inline README.md:83]: As I said [here](https://discord.com/channels/1161389681842343997/1174250586758201355/1175111183251017799) this does not work

File: db_entries/init.sql
Comment (PolimiGiovanniArriciati) [inline db_entries/init.sql:1]: ?

**Review by PolimiGiovanniArriciati — CHANGES_REQUESTED** [review]

**Review by PolimiGiovanniArriciati — DISMISSED** [review]
For me now we are good, If it's okay for you merge the branch

**Review by PolimiGiovanniArriciati — APPROVED** [review]

File: db_entries/init.sql [hunk 4]
```diff
@@ -27,6 +27,15 @@ CREATE TABLE IF NOT EXISTS CLIENT_ALLERGY
...
+
+--MOCK DATA
+INSERT INTO CLIENT VALUES(2, 'M', 'MARCOS PETRUCCI', 23, 'PASTA AL POMODORO');
+INSERT INTO FOOD_ALLERGY VALUES(1, 'SHRIMP');
+INSERT INTO FOOD_ALLERGY VALUES(2, 'PEANUT');
+INSERT INTO CLIENT_ALLERGY VALUES(2,1);
+INSERT INTO CLIENT_ALLERGY VALUES(2,2);
+INSERT INTO CLIENT VALUES(3, 'M', 'GIOVANNI A', 23, 'PIZZA');
+
 INSERT INTO CLIENT (GENDER, NAME, AGE, FAVORITE_FOOD) VALUES('M', 'MARCOS PETRUCCI', 23, 'PASTA AL POMODORO');
```
Comment (febagni) [inline db_entries/init.sql:39]: repeated line

Comment (febagni) [reply db_entries/init.sql:39]: must be removed

**Review by febagni — CHANGES_REQUESTED** [review]

**Review by febagni — APPROVED** [review]
Looks good to me

**Pull request merged**